src/components/controller/leftController/leftController.js

```diff
@@ -0,0 +1,23 @@
+import { createElement } from "../../../utils/elements";
+import triangleSrc from "../../../assets/triangle.svg";
+export const createLeftController = () => {
+  const sectionLeftController = createElement("section", {
+    className: "leftController",
+  });
+  const lamp = createElement("div", { className: "lamp" });
+  sectionLeftController.append(lamp);
+  const keys = createElement("div", { className: "keys" });
+  sectionLeftController.append(keys);
+
+  const buttons = ["btn-up", "btn-left", "btn-right", "btn-down"];
```

Comment:
I would expect to have an array of elements instead of classes when reading `buttons`.

```suggestion
  const buttonClassNames = ["btn-up", "btn-left", "btn-right", "btn-down"];
```